pages/common/qmv.md

```diff
@@ -0,0 +1,24 @@
+# qmv
+
+> Move files and directories with your favorite text editor.
+> More information: <https://www.nongnu.org/renameutils/>.
+
+- Move a single file (invoke an editor with source on the left, target on the right):
+
+`qmv {{source_file}}`
+
+- Move multiple JPG files:
+
+`qmv {{*.jpg}}`
+
+- Move directories:
+
+`qmv --directory {{my_directory/}} {{another_directory/}} {{the_other_directory/}}`
```

Comment:
I personally like short option too! But since I never contribute to TLDR before, I read the guild line and see that long option is more preferable. So I PR the long option.

I've change it to short option in 166264880efc5de6535bc741e6df7f4f10e5020f since I recall that `ls -d` also work the same way (and the short option is already de facto standard in this case, I think).

I also change `my_directory` to just `directory`.